<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SmvAbnormalQualitySet</class>
 <widget class="QDialog" name="SmvAbnormalQualitySet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>555</width>
    <height>317</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>品质参数设置</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_3" columnstretch="1,1">
     <item row="0" column="0">
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>设置</string>
       </property>
       <layout class="QGridLayout" name="gridLayout" columnstretch="2,2,1">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>有效[bit0-1]</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" colspan="2">
         <widget class="QScrollComboBox" name="cmb_bit1">
          <item>
           <property name="text">
            <string>00：好</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>01：无效</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>10：保留</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>11：可疑</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>源[bit10]</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" colspan="2">
         <widget class="QScrollComboBox" name="cmb_bit10">
          <item>
           <property name="text">
            <string>0：过程</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1：被取代</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>测试[bit11]</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1" colspan="2">
         <widget class="QScrollComboBox" name="cmb_bit11">
          <item>
           <property name="text">
            <string>0：运行</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1：测试</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="4" column="1">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>110</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>操作员闭锁[bit12]</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1" colspan="2">
         <widget class="QScrollComboBox" name="cmb_bit12">
          <item>
           <property name="text">
            <string>0：不闭锁</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1：闭锁</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>详细设置</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2" columnstretch="0,0">
        <item row="0" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>溢出[bit2]</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QScrollComboBox" name="cmb_bit2">
          <item>
           <property name="text">
            <string>0：无溢出</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1：溢出</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>超值域[bit3]</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QScrollComboBox" name="cmb_bit3">
          <item>
           <property name="text">
            <string>0：正常</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1：超值域</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>坏基准值[bit4]</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QScrollComboBox" name="cmb_bit4">
          <item>
           <property name="text">
            <string>0：正常</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1：坏基准值</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>抖动[bit5]</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QScrollComboBox" name="cmb_bit5">
          <item>
           <property name="text">
            <string>0：无抖动</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1：有抖动</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>故障[bit6]</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QScrollComboBox" name="cmb_bit6">
          <item>
           <property name="text">
            <string>0：无故障</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1：有故障</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>旧数据[bit7]</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QScrollComboBox" name="cmb_bit7">
          <item>
           <property name="text">
            <string>0：无超时</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1：数据超时</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>不一致[bit8]</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QScrollComboBox" name="cmb_bit8">
          <item>
           <property name="text">
            <string>0：一致</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1：不一致</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>不精确[bit9]</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QScrollComboBox" name="cmb_bit9">
          <item>
           <property name="text">
            <string>0：精确</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1：不精确</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QSttCheckBox" name="checkBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>修改全部通道值</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QScrollComboBox" name="cmb_DispType">
       <item>
        <property name="text">
         <string>二进制</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>十进制</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>十六进制</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QSttLineEdit" name="lineEdit"/>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbn_OK">
       <property name="text">
        <string>确认</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbn_Cancle">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QSttLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">../../../Module/UI/Controls/SttLineEdit.h</header>
  </customwidget>
  <customwidget>
   <class>QSttCheckBox</class>
   <extends>QCheckBox</extends>
   <header>../../Controls/SttCheckBox.h</header>
  </customwidget>
  <customwidget>
   <class>QScrollComboBox</class>
   <extends>QComboBox</extends>
   <header>../../Module/ScrollCtrl/ScrollComboBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
